word-guess.rb

```diff
@@ -0,0 +1,131 @@
+require "colorize"
+require 'random_word_generator'
+
+class Word
```

Comment:
I like that you've split `Word` out into a separate class here. This does a good job of separating game logic from the code to display a word, making the whole program more clear.